### PR TITLE
singularity - added master branch

### PR DESCRIPTION
**Pull request by JusticeForMikeBrown**
Centos 6 needs branch per [this issue I opened](https://github.com/singularityware/singularity/issues/1056)

### CODE REVIEW

File: var/spack/repos/builtin/packages/singularity/package.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ class Singularity(AutotoolsPackage):
 
     version('2.4', 'd357ce68ef2f8149edd84155731531465dbe74148c37719f87f168fc39384377')
     version('2.3.1', '292ff7fe3db09c854b8accf42f763f62')
+    version('master', git='https://github.com/singularityware/singularity.git')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/singularity/package.py]: please call it `develop` to be consistent with the rest of Spack

**Comment by JusticeForMikeBrown** [comment]
Done!

**Comment by JusticeForMikeBrown** [comment]
Hi

Can we please merge?  No idea why Travis CI failed unit test.

**Comment by adamjstewart** [comment]
> No idea why Travis CI failed unit test.

Me neither. I restarted your Travis tests. Let me know when it passes and I'll merge.

**Pull request merged**